src/styles/blocks/card.scss

```
@@ -0,0 +1,95 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  text-align: center;
+  transition: transform 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 36px;
+  }
+
+  &__title {
+    display: inline-block;
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: start;
+    transition: color 0.3s;
+
+    .card:hover & {
+      color: #34568b;
+    }
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+    text-align: start;
+    font-weight: 400;
```

Default value, so you can just remove it everywhere
```suggestion
    text-align: start;
```